static/src/javascripts/lib/capture-perf-timings.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import ophan from 'ophan/ng';
+import { getMarkTime } from 'lib/user-timing';
+import performanceAPI from 'lib/window-performance';
+
+const capturePerfTimings = (): void => {
```

Comment:
I agree in principle but I think we are going for full conversion to arrow functions where possible.